controllers/dbaasprovider_controller.go

```diff
@@ -177,7 +177,7 @@ func bridgeProviderCR(clusterRoleList *rbac.ClusterRoleList) *dbaasoperator.DBaa
 		},
 
 		Spec: dbaasoperator.DBaaSProviderSpec{
-			Provider: dbaasoperator.DatabaseProvider{
+			Provider: dbaasoperator.DatabaseProviderInfo{
```

Comment:
Add GroupVersion to indicate that RDS is using v1beta1 of dbaas API/data structure?